modules/optimization/examples/bimaterial_mesh/main.i

```diff
@@ -0,0 +1,117 @@
+# This performs the same 2 parameter inversion problem done
+# in test/tests/optimizationreporter/bimaterial using the
+# the ParameterMeshOptimization OptimizationReporter
+#
+[Optimization]
+[]
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 16
+    ny = 16
+    xmin = -4
+    xmax = 4
+    ymin = -4
+    ymax = 4
+  []
+[]
+
+[OptimizationReporter]
+  type = ParameterMeshOptimization
+  parameter_names = 'diffusivity_values'
+  parameter_meshes = parameter_mesh_in.e
+  num_parameter_times = 1 # this is the default value
+  parameter_families = MONOMIAL
+  parameter_orders = CONSTANT
+  initial_condition = '3' # Should this be per paramter... maybe restarted from mesh
```

Comment:
That is correct.  At some point we need to change this so that each individual parameter can get an initial condition that is read from the mesh, this would allow us to restart an optimization problem or do parameter refinement from a a coarse mesh optimization.